src/components/MermaidDash/MermaidDash.jsx

```diff
@@ -40,6 +40,7 @@ const MermaidDash = ({ isApiDataDoneLoading, setIsApiDataDoneLoading }) => {
   const { isLoading, isAuthenticated, getAccessTokenSilently } = useAuth0()
   const [loadedProjectsCount, setLoadedProjectCount] = useState(0)
   const [totalProjectsCount, setTotalProjectsCount] = useState(0)
+  const [showLoadingIndicator, setShowLoadingIndicator] = useState(!isApiDataDoneLoading)
```

Comment:
I hastily had removed the styling which moves the metrics on mobile up to make room for the loading indicator. I added it back and fixed it up a little. It required this state to live in a parent.

Reply:
`isApiDataDoneLoading` is a bit of a mouthful. Verging into personal preferences here but I prefer something like `apiDataLoaded`

Reply:
DEf a mouthful. Lol, Ill meet you in the middle with `isApiDataLoaded` to keep the `is` for 'proper' boolean naming. 

Even 'api' here is vague, but Im not fully sure on if its 'projects' data and thats what we should call it being new to the project and data.